src/i965_drv_video.c

```diff
@@ -1278,9 +1291,6 @@ i965_CreateConfig(VADriverContextP ctx,
                     attrib.value = 1;
                 else
                     attrib.value = I965_MAX_NUM_SLICE;
-            } else if (profile == VAProfileHEVCMain ||
-                       profile == VAProfileHEVCMain10) {
-                attrib.value = 1;
```

Comment:
No, it can't be removed even if avc multi-slices is ready on SKL+.  intel_enc_hw_context_init() will check whether VAConfigAttribEncMaxSlices is included in the config attribute list.